submissions/Lu-sure/dom-api/style.css

```diff
@@ -0,0 +1,112 @@
+* {
+    padding: 0;
+    margin: 0;
+    font-size: 22px;
+    box-sizing: border-box;
+    font-family: Comfortaa;
+}
+
+html, body {
+    width: 100%;
+    height: 100%;
+}
+
+.wrapper {
+    display: flex;
+}
+
+.side{
+    min-width: 300px;
+    background-color: rgb(37, 49, 116);
+    height: auto;
+}
+
+.side.open {
+    transform: translate(0, 0);
+}
+
+.main {
+    max-height: 1200px;
+}
+
+.side-hero {
+    height: 50px;
+    width: 280;
+    margin: 20px 10px; 
+    padding: 10px;
+    background-color: rgb(201, 187, 221);
```

Comment:
Same as with height in side. When content changes - block change height. Maybe add background color to main ?)
Because it is just visual improvements, iit is up to you

Reply:
Fixed.